paper.md

```diff
@@ -76,19 +76,21 @@ Finally, as both data management plan and data protection impact assessment are
 
 ## Creating a DPIA Knowledge Model
 
-*TODO: (Pinar), Vilém, Nene, and Paulette - what was the process, how you worked, what is the result*
+The DPIA knowledge model is mainly inspired by the several existing documents, namely the [CNIL DPIA document template](https://www.cnil.fr/sites/default/files/atoms/files/cnil-pia-2-en-templates.pdf), the DPIA document template used by the Luxembourg Centre for Systems Biomedicine and [ICO guidance](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/data-protection-impact-assessments-dpias/how-do-we-do-a-dpia/#how)) on data protection.
 
-The DPIA knowledge model aims to help in managing the data protection risks of a research project by identifying and analyzing those risks and the solutions implemented to mitigate them. This knowledge model is mainly inspired by the [CNIL Template](https://www.cnil.fr/sites/default/files/atoms/files/cnil-pia-2-en-templates.pdf), the DPIA template from the Luxembourg Centre for Systems Biomedicine (LCSB) DPIA Template, and some [ICO guidance](https://ico.org.uk/for-organisations/guide-to-data-protection/guide-to-the-general-data-protection-regulation-gdpr/data-protection-impact-assessments-dpias/how-do-we-do-a-dpia/#how)). The DPIA knowledge model describes:
+The first chapter collects administrative information and description of method followed for during project's DPIA. Following chapters contain questions related to:
```

Comment:
The first chapter collects administrative information and description of the method(s) followed to establish the project's DPIA. Following chapters contain questions related to: